diff --git a/vendor/github.com/gen2brain/raylib-go/raylib/cgo_linux_drm.go b/vendor/github.com/gen2brain/raylib-go/raylib/cgo_linux_drm.go
index afc1ac0..cc27425 100644
--- a/vendor/github.com/gen2brain/raylib-go/raylib/cgo_linux_drm.go
+++ b/vendor/github.com/gen2brain/raylib-go/raylib/cgo_linux_drm.go
@@ -5,6 +5,6 @@ package rl
 
 /*
 #cgo linux,drm LDFLAGS: -lGLESv2 -lEGL -ldrm -lgbm -lpthread -lrt -lm -ldl
-#cgo linux,drm CFLAGS: -DPLATFORM_DRM -DGRAPHICS_API_OPENGL_ES2 -DEGL_NO_X11 -I/usr/include/libdrm
+#cgo linux,drm CFLAGS: -DPLATFORM_DRM -DGRAPHICS_API_OPENGL_ES2 -DEGL_NO_X11 
 */
 import "C"
