--- a/meson.build	2024-08-30 20:40:50.009081663 -0500
+++ b/meson.build	2024-08-30 20:41:43.789425652 -0500
@@ -295,7 +295,6 @@
   'include/drm/savage_drm.h', 'include/drm/sis_drm.h',
   'include/drm/tegra_drm.h', 'include/drm/vc4_drm.h',
   'include/drm/via_drm.h', 'include/drm/virtgpu_drm.h',
-  subdir : 'libdrm',
 )
 if with_vmwgfx
   install_headers('include/drm/vmwgfx_drm.h', subdir : 'libdrm')
